components/Nav/Nav.tsx

```diff
@@ -62,7 +67,17 @@ export const Nav = (): JSX.Element => {
               <DrawerHeader>
                 <DrawerTitle data-testid="title">Gridiron Survivor</DrawerTitle>
               </DrawerHeader>
-              <ul className="m-0 flex flex-col gap-4 p-0">
+              <ul className="m-0 flex flex-col gap-1 p-0">
```

Comment:
What was the reasoning behind changing this from gap-4 to gap-1?

Reply:
@choir27  To make the gap between settings and logout smaller.